.github/workflows/ci.yml

```diff
@@ -389,7 +389,7 @@ jobs:
           brew install diffutils
 
       - name: Test
-        timeout-minutes: 20
+        timeout-minutes: 30
```

Comment:
Do macOS tests take more than 20 minutes?

Reply:
The macos-14 test took approximately 18 minutes, the macos-13 test took approximately 19 minutes. If the flaky test was retried it took over 20 minutes.

Reply:
I see. (It might be better to increase the time out only on macOS, though.)

Reply:
Yes, CI jobs are divided into linux, windows, and macos, so this change is only for macos.

Reply:
Ah, I overlooked that.